src/classes/ERR_Notifier_TEST.cls

```diff
@@ -51,10 +53,189 @@ private class ERR_Notifier_TEST {
+    * returned list of email recipients consists of all System Admin users
+    */
+    @IsTest
+    private static void shouldReturnAllSystemAdminsEmailsIfErrorUserIsInactive() {
+        // Create a System Admin User
+        User tempUser = UTIL_UnitTestData_TEST.createUserWithoutInsert(UTIL_Profile.SYSTEM_ADMINISTRATOR);
+        tempUser.IsActive = true;
+        insert tempUser;
+
+        Id fakeUserId = UTIL_UnitTestData_TEST.mockId(User.SObjectType);
+        Error_Settings__c errorSettings = UTIL_CustomSettingsFacade.getErrorSettingsForTests(
+            new Error_Settings__c(
+                Error_Notifications_On__c = True,
+                Error_Notifications_To__c = fakeUserId
+            )
+        );
+        String errorNotifRecipient = errorSettings.Error_Notifications_To__c;
+
+        Set<String> expectedEmails = getSystemAdminEmails();
+
+        Test.startTest();
+        ERR_Notifier notifierService = new ERR_Notifier();
+        List<String> errorRecipients = notifierService.getErrorEmailRecipients(errorNotifRecipient);
+        Test.stopTest();
+
+        System.assert(errorRecipients.size() > 1, 'There should be more than one System Admin user listed');
+        System.assert(expectedEmails.containsAll(errorRecipients), 'All System Admins should be set as recipients');
+    }
+
+    /**********************************************************************************************
+    * @description Verify that if the specified Error Notification User is inactive (or doesn't exist), that the
+    * generated email includes a note about it.
+    */
+    @IsTest
+    private static void shouldIncludeNoteAboutInactiveNotificationUserInEmail() {
+        Id fakeUserId = UTIL_UnitTestData_TEST.mockId(User.SObjectType);
+        Error_Settings__c errorSettings = UTIL_CustomSettingsFacade.getErrorSettingsForTests(
+            new Error_Settings__c(
+                Error_Notifications_On__c = True,
+                Error_Notifications_To__c = fakeUserId
+            )
+        );
+        String errorNotifRecipient = errorSettings.Error_Notifications_To__c;
+
+        Test.startTest();
+        ERR_Notifier notifierService = new ERR_Notifier();
+        List<String> errorRecipients = notifierService.getErrorEmailRecipients(errorNotifRecipient);
+        Messaging.SingleEmailMessage emailMessage = notifierService.createEmailMessage(null, new List<Error__c>(), errorRecipients);
+        Test.stopTest();
+
+        System.assert(emailMessage.getPlainTextBody().containsIgnoreCase('not an active user'));
+        System.assert(emailMessage.getPlainTextBody().containsIgnoreCase(errorNotifRecipient));
+    }
+
+    /**********************************************************************************************
+    * @description Verify that if the specified Error Notification User is inactive (or doesn't exist), that the
+    * generated email includes a note about it.
+    */
```

Comment:
Looks like this description was copied from the positive case, it should say that if the user is active that the email should NOT include a note about it

Reply:
Good catch. Fixed.